contracts/WakuRlnRegistry.sol

```diff
@@ -67,12 +67,12 @@ contract WakuRlnRegistry is Ownable {
         }
     }
 
-    function register(uint16 storageIndex, uint256[] calldata commitments) external payable {
+    function register(uint16 storageIndex, uint256[] calldata commitments) external {
         if (storageIndex >= nextStorageIndex) revert NoStorageContractAvailable();
         WakuRln(storages[storageIndex]).register(commitments);
     }
 
-    function register(uint16 storageIndex, uint256 commitment) external payable {
```

Comment:
Any reason these have been `payable` in the first place?

Reply:
yes, they were payable since we may want to allow WakuRln storage implementations to use it, however, after discussion, scope has been cut re: using eth as a payment method, hence deprecated here